.github/workflows/build-package-docs.yaml

```
@@ -160,7 +150,7 @@ jobs:
       if: env.TARGET == 'production'
       env:
         BRANCH_NAME: ${{ github.event.inputs.ansible-package-version }}
-        PROD_URL: https://ansible.readthedocs.io/projects/ansible/
+        PROD_URL: https://ansible.readthedocs.io/projects/ansible
```

How about 
```suggestion
        PROD_URL: https://ansible.readthedocs.io/projects/ansible/${{ github.event.inputs.ansible-package-version }}/
```
? Or is it concatenated elsewhere?

---

Oh.. I see there's `ENV_URL` that's used in the `environment:` setting — does that actually work? I'm pretty sure it's computed too late and the GH UI is probably showing nothing.

---

Oh, interesting… it does show up in https://github.com/ansible/ansible-documentation/actions/runs/9566619506. But with a double slash. Now I understand why you'd want to drop it.

Marking the thread as resolved.